form.py

```diff
@@ -0,0 +1,59 @@
+from flask_wtf import Form
+from wtforms import StringField, BooleanField, SelectField, TextAreaField, RadioField
+from flask_wtf.file import FileField
+from wtforms.validators import DataRequired, Email, URL
+from werkzeug import secure_filename
+
+class ApplicationForm(Form):
+    name = StringField('Name', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(),Email()])
+    phone = StringField('Phone')
+    city = StringField('City')
+    state = StringField('State')
+    zipcode = StringField('Zip Code')
+    citizenship = BooleanField('Verify you are a U.S. Citizen or Green Card holder')
+    primary_skillset = SelectField('Please indicate your primary skillset. Your response here will be used primarily in assigning reviewers for your application.', choices=[
+           ('acquisition_specialist', 'Acquisition Specialist'),
```

Comment:
I went ahead and standardized the capitalization on these in 3221a763139c9b8f00b9ba3484661fe2991344a9.